src/reps/promise.js

```diff
@@ -41,13 +45,30 @@ const PromiseRep = React.createClass({
       keys.push("value");
     }
 
+    let {
+      attachedNodeFrontsByActor,
+      onDOMNodeMouseOver,
+      onDOMNodeMouseOut,
+      onInspectIconClick,
+    } = this.props;
+
     return keys.map((key, i) => {
+      let object = promiseState[key];
       return PropRep(Object.assign({}, this.props, {
         mode: MODE.TINY,
         name: `<${key}>`,
-        object: promiseState[key],
+        object,
         equal: ": ",
-        delim: i < keys.length - 1 ? ", " : ""
+        delim: i < keys.length - 1 ? ", " : "",
+        attachedNodeFrontsByActor,
+        attachedNodeFront: (
```

Comment:
attachedNodeFront is not part of the declared propTypes for PropRep and looking at the code of PropRep it doesn't seem to be used.